ebos/ecltracermodel.hh

```diff
@@ -385,24 +382,25 @@ protected:
                                           Dune::ForwardCommunication);
     }
 
-    template <class TrRe>
-    void updateStorageCache(TrRe & tr)
+    void updateStorageCache()
     {
-        if (tr.numTracer() == 0)
```

Comment:
Moving this check inside the loop, will it not cause iteration over the grid, even when there is no tracer? I think we should have a check here to avoid the iteration if there are no tracers as all.

Reply:
it's done on the outside, see https://github.com/OPM/opm-simulators/blob/master/ebos/ecltracermodel.hh#L162